### DISCUSSION TITLE
[站点背景] 为移动端和桌面端分别设定封面背景

### DISCUSSION THREAD

**Discussion by Leev1s** [tangly1024/NotionNext#2067] [Ideas]
**为什么提出这个新的特性改动**
hexo在移动端显示的封面背景被奇怪的放大了，效果不佳

**描述一下你推荐的解决方案**
分别设置一下可以提高用户体验

**描述一下你考虑过的其它替代解决方案**
选用纯色背景

**补充说明**
如果已有相关设定，🙏抱歉，还请告知，是我粗心了

**Comment by hfoi589** [comment]
支持
